docs/docs-content/clusters/edge/site-deployment/deploy-private-registry.md

```diff
@@ -0,0 +1,96 @@
+---
+sidebar_label: "Deploy Cluster with a Private Registry"
+title: "Deploy Cluster with a Private Registry"
+description: "Instructions for creating an Edge Native Cluster Profile"
+hide_table_of_contents: false
+sidebar_position: 60
+tags: ["edge"]
+---
+
+Palette Edge supports authentication with private image registries, which allows your cluster to pull images from a private registry during deployment. You can configure your cluster to pull images from a private registry for both cluster creation and cluster repave. To configure a cluster to pull images from a private image registry, provide the registry URL and the credentials needed to authenticate with the registry in the cluster profile. 
+
+:::caution
+If you have specified registry credentials in the `registryCredentials` field in the user data file during the EdgeForge process, the credentials provided in the cluster profile will be ignored.  For more information, refer to [EdgeForge - Build Artifacts](../edgeforge-workflow/palette-canvos.md).
+:::
+
+## Limitations
+
+- A cluster cannot pull images from more than one private registry.
+
+- If your private registry has TLS enabled, you can only configure a new cluster to use a TLS certificate with a private registry. You cannot configure an existing cluster with a TLS certificate to communicate with your private registry. 
+
+- Palette Edge supports basic username/password authentication. Token authentication schemes used by services such as AWS ECR and Google Artifact Registry are not supported. 
+
+## Prerequisites
+
+- At least one Edge host with an x86_64 or AMD64 processor architecture. 
+
+- A private image registry.
+
+- A provider image you created in the EdgeForge process stored in your private image registry. For more information, refer to [Build Artifacts](../edgeforge-workflow/palette-canvos.md).
+
+## Enablement
+
+1. Log in to [Palette](https://console.spectrocloud.com).
+
+2. Navigate to the left **Main Menu** and select **Profiles**.
+
+3. If you already have a cluster profile you want to deploy the cluster with, select that profile and select **Create new version** to create a new version of the profile to save your changes.
+
+   Otherwise, click **Add new profile** to create a new cluster profile. 
+
+4. Select the OS layer of your cluster profile. If you are creating a new profile, you will get to configuring the OS layer after filling out **Basic Information** and **Cloud Type**. 
+
+5. Update the `system.uri` parameter in the pack editor for your OS layer. Use the custom OS image you created in the EdgeForge process. Refer to the EdgeForge [Build Images](../edgeforge-workflow/palette-canvos.md) guide if you are missing a custom OS image. The following is an example configuration using a custom OS image.
```

Comment:
```suggestion
5. Update the `system.uri` parameter in the pack editor for your OS layer. Use the custom OS image you created in the EdgeForge process. Refer to the EdgeForge [Build Images](../edgeforge-workflow/palette-canvos.md) guide if you are missing a custom OS image. The following is an example configuration using the Bring Your Own Operating System (BYOOS) pack with a custom OS image.
```

Might be good to clarify this is the BYOOS pack and link to the BYOOS pack page
https://docs.spectrocloud.com/integrations/byoos